pkgs/applications/networking/mailreaders/hey-mail/default.nix

```diff
@@ -0,0 +1,121 @@
+{ fetchurl, lib, stdenv, squashfsTools, makeWrapper, c-ares, gtk3-x11,
+glib, libevent, libdrm, libvpx, libxslt, libnotify,
+libappindicator-gtk2, libappindicator-gtk3, atk, mesa, cups, systemd,
+alsaLib, at-spi2-atk, at-spi2-core, gdk-pixbuf, gnome2, cairo, xorg,
+ffmpeg, http-parser, nss, nspr, dbus, expat }:
+
+let
+  deps = [
+    c-ares
+    gtk3-x11
+    glib
+    libevent
+    libdrm
+    libvpx
+    libxslt
+    libnotify
+    libappindicator-gtk2
+    libappindicator-gtk3
+    atk
+    mesa
+    cups
+    systemd
+    alsaLib
+    at-spi2-atk
+    at-spi2-core
+    gdk-pixbuf
+    gnome2.pango
+    cairo
+    xorg.libxcb
+    xorg.libX11
+    xorg.libXcomposite
+    xorg.libXcursor
+    xorg.libXdamage
+    xorg.libXext
+    xorg.libXfixes
+    xorg.libXi
+    xorg.libXrender
+    xorg.libXtst
+    xorg.libXrandr
+    ffmpeg
+    http-parser
+    nss
+    nspr
+    dbus
+    expat
+    stdenv.cc.cc
+  ];
+
+  version = "1.1.0";
+  rev = "10";
+
+in
+
+stdenv.mkDerivation rec {
+  pname = "hey-mail";
+  inherit version;
+
+  src = fetchurl {
+    url = "https://api.snapcraft.io/api/v1/snaps/download/lfWUNpR7PrPGsDfuxIhVxbj0wZHoH7bK_${rev}.snap";
+    sha256 = "4fbd1f4cca1bfe2bd0c57c5df187459f2ce53cf5fd66f8ea155de3b810bfc7ce";
+  };
+
+  nativeBuildInputs = [
+    squashfsTools
+    makeWrapper
+  ];
+
+  dontStrip = true;
+  dontPatchELF = true;
+
+  unpackPhase = ''
+  runHook preUnpack
+  unsquashfs "$src"
+  cd squashfs-root
+  runHook postUnpack
+  '';
+
+  installPhase = ''
+  runHook preInstall
+
+  mkdir -p $out/lib
+  mv ./* $out/
+
+  cp $out/meta/snap.yaml $out
```

Comment:
Don't we have that file doubled in the output? ln -s is probably preferred here.